source/guides/clear/mixer.rst

```diff
@@ -411,27 +479,197 @@ mix version 10 (from which the image was built), to mix version 20.
    .. code-block:: bash
 
       poweroff
 
+Example 6: Remove a bundle
+==========================
+
+Removing a bundle requires more work than deleting the bundle description
+file, as shown in Example 2. After a bundle is built in the mix, you must
+assure all of the files that are part of the bundle are removed from the
+client where that bundle is installed. To do this, create a version of this
+bundle in which all of its content is marked for deletion.
+
+In the following example, we show how to remove the contents of the `editors`
+bundle that we added to our mix in Example 2.
+
+#. First update your mix version. This will set the mix to the next version.
+
+   .. code-block:: bash
+
+      mixer versions update
+
+   .. note::
+      Run this command every time that you want to build a new version.
+
+#. Navigate to local-bundles:
+
+   .. code-block:: bash
+
+      cd local-bundles
+
+#. Use your preferred editor to open the `editors` bundle.
+
+   .. code-block:: bash
+
+      nano editors
+
+#. Delete **all lines** that follow after the `[MAINTAINERS]` line.
+
+#. After, your file will appear as follows:
+
+   .. code-block:: console
+
+      # [TITLE]: editors
+      # [DESCRIPTION]: Run popular terminal text editors.
+      # [STATUS]: Active
+      # [CAPABILITIES]:
+      # [TAGS]: Tools and Utilities, Editor
+      # [MAINTAINER]: Developer <developer@intel.com>
+
+#. Save and exit.
+
+#. Next run a build to capture recently edited bundles and update your mix.
+
+   .. code-block:: bash
+
+      mixer build all
+
+   .. note::
+      :command:`mixer build all` runs both :command:`mixer build bundles` and :command:`mixer build update` in one step.
+
+At this point the new mixer version is complete. All the content of the
+editors bundles is marked as deleted. If any clients of this mix want to
+upgrade to mixer build version 20, the editors bundle’s content will have
+been removed. Note that the bundle still exists and is being tracked by
+swupd, but it contains no files. To learn how to remove the bundle from
+being listed and tracked by swupd, see **the following example**.
+
+Example 7: Execute a format bump
+================================
+
+As a maintainer of your mix, you must execute a format bump if you wish to:
+* Track upstream’s format bump on your downstream derivative
+* Delete any custom bundles that were added
+
+Follow the appropriate use case below depending on your needs.
+
+Basic
+-----
+
+If you maintain your own downstream derivative and you want to track
+upstream, you need to do a format bump when one occurs on upstream. This
+method helps you track the latest changes on upstream; however, it does not
+change any local content that was added or deleted. For example, if you
+deprecated bundles, this method will **not remove the bundle tracking**.
+Refer to `Advanced`_ for help on managing your local mix and removing bundle
+tracking.
+
+In this example, we show a mix version that was initialized to upstream
```

Comment:
Recommend lines 588 - 590 as follows:

"In this example, we show a mix version that was based on upstream
version 29740 (format 27). You need to update your mix to upstream version 30700 (format 28).
To do so, you will go through a format bump."